style.css

```diff
@@ -0,0 +1,203 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(images/Header.svg);
+  background-repeat: no-repeat;
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
```

Comment:
- [ ] As requester by the previous reviewer, the background position and margins are still not the same as the original design. Kindly check and ensure that the background image streches and covers the whole headline section as shown in the figma design. 
- [ ] The hamburger menu is also not displaying because you have put a` display: none` on the container that contains the hamburger menu. Kindly check and fix it. :wink: 


### Your design 

![Screenshot from 2023-02-17 22-26-01](https://user-images.githubusercontent.com/65233860/219805350-27237c65-2a5a-4be3-86fc-a704083438a3.png)



### Figma design 

![Screenshot from 2023-02-17 22-27-44](https://user-images.githubusercontent.com/65233860/219805472-b094b319-9567-4301-acd1-9d0ffc9172dc.png)